data.lua

```diff
@@ -292,7 +292,7 @@ turn_to_tank =
         {
           priority = "low",
           width = 104,
-          height = 84,
+          height = 64,

```

Comment:
I think `height` is 83, because the height of `tank-base-mask-1.png` is 1826, and `height_in_frames` is 22.

Reply:
That does seem to make sense